Phonebook.samggannon/Phonebook.samggannon/Services/ContactsService.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Phonebook.samggannon.Controllers;
+using Phonebook.samggannon.Models;
+using Phonebook.samggannon.Utilities;
+using Spectre.Console;
+
+namespace Phonebook.samggannon.Services;
+
+internal class ContactsService
+{
+    #region Insert
+    internal static void AddContact()
+    {
+        var contact = new Contact();
+        contact.Name = AnsiConsole.Ask<string>("Enter a name for your contact *Required: ").Trim();
+        contact.PhoneNumber = GetPhoneNumber();
+        contact.Email = GetEmailInformation(contact);
+
+        ContactsController.AddContact(contact);
+    }
+
+    private static string? GetPhoneNumber()
+    {
+        string contactsPhoneNumber = "";
+        string rawPhoneNumber = AnsiConsole.Ask<string>("Enter a phone number for your contact *Required: ").Trim();
+        bool isValidPhoneNumber = Validation.IsPhoneNumberValid(rawPhoneNumber);
+
+        while(!isValidPhoneNumber)
+        {
+            AnsiConsole.WriteLine("Invalid phone number. Please provide country and area code.");
+            AnsiConsole.WriteLine("Phone number must be at least 10 digits");
+            AnsiConsole.WriteLine("");
```

Comment:
The user feedback is not entirely correct. The phone number must be 10 digits for it to be accepted.